Nexus/Include/Nexus/MarketDataService/MarketOrderImbalanceExpression.hpp

```diff
@@ -30,20 +30,20 @@ namespace MarketDataService {
   class MarketOrderImbalanceExpression :
       public Beam::Expressions::IterationExpression<OrderImbalance> {
```

Comment:
Could not find `Beam::Expressions` either, so this file does not really compile.

Reply:
This is also dead code, delete it.